.github/workflows/build.yml

```diff
@@ -16,13 +16,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-13, windows-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
         configuration: [Release]
         include:
           - os: ubuntu-latest
             COMMAND: AM2RLauncher.Gtk -p:PublishSingleFile=true -p:DebugType=embedded -r ubuntu.18.04-x64 --no-self-contained
```

Comment:
linux fails here:
> The specified RuntimeIdentifier 'ubuntu.18.04-x64' is not recognized.

Reply:
Looks like the existing runtime is not one of the [listed runtime identifiers](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids) - I'll add a commit to fix it.

Reply:
oh wow, i didnt notice that they cut off like half their RIDs